src/utils/setup/create-podcast-pages.ts

```diff
@@ -0,0 +1,25 @@
+import path from 'path';
+import { getTagPageUriComponent } from '../get-tag-page-uri-component';
+import { fetchBuildTimeMedia } from '../../utils/setup/fetch-build-time-media';
+import { Podcast } from '../../interfaces/podcast';
+
+export const createPodcastPages = async (
+    createPage: Function,
+    allPodcasts: Podcast[],
+    metadata: object
+) => {
+    allPodcasts.map((podcast: Podcast) => {
```

Comment:
```suggestion
    allPodcasts.forEach((podcast: Podcast) => {
```

This is a nitpick, but since we really are not using the returned array from the map, `forEach` is a more clean function to use here.